contracts/express/AxelarExpressExecutable.sol

```diff
@@ -12,12 +12,12 @@ contract AxelarExpressExecutable is ExpressExecutorTracker {
     using SafeTokenTransfer for IERC20;
     using SafeTokenTransferFrom for IERC20;
 
-    IAxelarGateway public immutable gateway;
+    IAxelarGMPGatewayWithToken public immutable gateway;
```

Comment:
```suggestion
    IAxelarGMPGatewayWithToken public immutable gatewayAddress;
```
change the name so it doesn't conflict with the public interface

Reply:
if you inherit the executable contract, then this can be removed

Reply:
inherited the `AxelarGMPExecutableWithToken`.